packages/plugin-node-surrounding-code/test/surrounding-code.test.js

```diff
@@ -0,0 +1,129 @@
+const { describe, it, expect } = global
+
+const plugin = require('../')
+const { join } = require('path')
+const Report = require('@bugsnag/core/report')
+const Client = require('@bugsnag/core/client')
+const VALID_NOTIFIER = { name: 't', version: '0', url: 'http://' }
```

Comment:
Good point. For a more memory-safe implementation we could use streams rather than buffering the entire source file in memory. It is possible for a very large file to result in an OOM crash.